frontend/app/controllers/gallery/edit.js

```diff
@@ -0,0 +1,14 @@
+import Controller from "@ember/controller"
+
+export default Controller.extend({
+  actions: {
+    editGallery() {
+      let editedTitle = this.get("model.title")
+      let gallery = this.get("model")
+      gallery.set("title", editedTitle)
+      gallery.save().then(() => {
+        this.transitionToRoute("gallery.index")
+      })
```

Comment:
This should be enough:

```javascript
editGallery() {
  this.get("model").save().then(() => {
    this.transitionToRoute("gallery.index")
  })
}
```

You set the value of `model.title` to the value which it already has :wink: